### DISCUSSION TITLE
marker naming standard

### DISCUSSION THREAD

**Discussion by ClayBirkett** [solgenomics/sgn#3485] [General]
Find method to identify identical markers in nd_protocolprop table. For GBS markers we have many identical markers across protocols

create materialized view of nd_protocolprop table that uses a standard naming method for markers or
add new entry to nd_protocolprop table with new type_id
species + assembly + chromosome + position + ref + alt

benifits
1. can creation of variant page that list identical markers (marker synonyms)
2. can identify markers from other sources (EnsemblPlant) that have similar markers
3. can combine markers to get consensus 
4. can combine markers analysis (GWAS) for meta analysis
5. query for marker names are much quicker

**Comment by ch728** [comment]
This seems like it would be useful. Would it make sense to have a separate variant ID and then marker IDs, which are specific to a given platform/protocol? Essentially variant ID is like the accession name and marker IDs are synonyms for a given variant ID. Then marker groups could be defined for subsetting, which are lists of marker IDs that define a QTL or whatever else a user wants. I think GOBii has a concept like this in their GDM. Having some way to subset data across protocols would be nice. My current work around is to define new protocols that represent an intersection of one or more other protocols.
